File: Python/course.py
class Course: # Deklarasi class Course
    __courseName = "" # Untuk menyimpan nama Mata Kuliah, atribut berjenis private

    def __init__(self, courseName = ""): # Membuat konstruktor
        self.__courseName = courseName # Menginisialisasi atribut __courseName

    def setCourseName(self, courseName): # Setter untuk atribut __courseName
        self.__courseName = courseName # Menginisialisasi atribut __courseName

    def getCourseName(self): # Getter untuk atribut __courseName
        return self.__courseName # Mengembalikan nilai atribut __courseName
    

File: C++/Course.cpp
// Import library yang dibutuhkan
#include <bits/stdc++.h>

using namespace std; // Menggunakan standard namespace

class Course // Deklarasi class Course
{
private:               // Membuat atribut berjenis private
    string courseName; // Untuk menyimpan nama mata kuliah

public:
    /* Konstruktor */
    Course() // Membuat konstruktor tanpa isian
    {
        this->courseName = "";
    }

    Course(string courseName) // Membuat konstruktor dengan isian dari parameter
    {
        this->courseName = courseName;
    }

    /* Setter dan Getter untuk setiap atribut di dalam class Course */
    void setCourseName(string courseName)
    {
        this->courseName = courseName;
    }

    string getCourseName()
    {
        return this->courseName;
    }

    // Destruktor
    ~Course()
    {
    }
};